### ISSUE TITLE
A Plague Tale: Innocence (Windows x64)

### ISSUE THREAD

**Issue by Firlaev-Hans** [ptitSeb/box86-compatibility-list#118]
### Description
Ingame but not playable due to bugs. Very demanding (2019 game).

### Tested on
NVIDIA Jetson Nano 4GB
Xubuntu 20.04
Wine 6 upstream
DXVK 1.5 (newer DXVK doesn't support D3D11 with the given Vulkan features)
box64 1.7 (284345d)

### Status
**Current behavior:**
The game launches and the menu is displayed correctly minus the mouse cursor.
I set the graphics to a minimum and resolution scaling to 30% which is why the screenshots look so incredibly blurry.
Starting a new game works and the game runs, but is not playable with mouse + keyboard because the cursor is not captured. A gamepad works better.
There are graphical glitches, but more importantly, it is not possible to play because you can walk through characters, and the first cutscene doesn't get triggered when you go to the father.

### Screenshots
![Menu](https://user-images.githubusercontent.com/61254500/142887001-d85876cb-efa5-4d2e-af40-9be3fab8c438.png)
![Ingame](https://user-images.githubusercontent.com/61254500/142887013-8e2cb50a-fdc0-4ca5-9b7c-8959e5e1e8fd.png)
Amicia's Hair is not supposed to glow white and pink.
![Ingame_WrongColors](https://user-images.githubusercontent.com/61254500/142887023-9494d919-234e-4857-9766-2f99706ebfcd.png)
The colors are off in this scene. Can't interact with the father, but I can walk right through him.

### Vulkan
DXVK 1.5 (D3D11 FL11) on Vulkan 1.1

**Comment by clort81** [comment]
> I set the graphics to a minimum and resolution scaling to 30%

How did you do this?  Is this an option in box86?

**Comment by Firlaev-Hans** [comment]
> > I set the graphics to a minimum and resolution scaling to 30%
> 
> How did you do this? Is this an option in box86?

No, it's just a feature that this game happens to support in its graphics options.
